setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = shellcheck_py
-version = 0.9.0.6
+version = 0.10.0
```

Comment:
the last element is our version and should remain

Reply:
Should it be 0.10.0.0 or 0.10.0.1?

Reply:
Changed it to 0.10.0.1.